SLHCUpgradeSimulations/Configuration/python/phase1TkCustoms.py

```diff
@@ -416,9 +428,10 @@ def customise_Reco(process,pileup):
     process.pixelTracks.FilterPSet.chi2 = cms.double(50.0)
     process.pixelTracks.FilterPSet.tipMax = cms.double(0.05)
     process.pixelTracks.RegionFactoryPSet.RegionPSet.originRadius =  cms.double(0.02)
-    process.templates.DoLorentz=False
-    process.templates.LoadTemplatesFromDB = cms.bool(False)
-    process.PixelCPEGenericESProducer.useLAWidthFromDB = cms.bool(False)
+    # use defaults d.k. 2/16
+    #process.templates.DoLorentz=False
+    #process.templates.LoadTemplatesFromDB = cms.bool(False)
+    #process.PixelCPEGenericESProducer.useLAWidthFromDB = cms.bool(False)

```

Comment:
Matti,
In this pull request there is also a modification to the
fakeConditions_Phase1_cff.py. The old version overwrites some DB conditions with fake.
I think this has to be modified.
Danek


On 10 Feb 2016, at 17:01, Matti Kortelainen <notifications@github.com<mailto:notifications@github.com>> wrote:


In SLHCUpgradeSimulations/Configuration/python/phase1TkCustoms.py<https://github.com/cms-sw/cmssw/pull/13228#discussion_r52477843>:

> @@ -416,9 +428,10 @@ def customise_Reco(process,pileup):
>      process.pixelTracks.FilterPSet.chi2 = cms.double(50.0)
>      process.pixelTracks.FilterPSet.tipMax = cms.double(0.05)
>      process.pixelTracks.RegionFactoryPSet.RegionPSet.originRadius =  cms.double(0.02)
> -    process.templates.DoLorentz=False
> -    process.templates.LoadTemplatesFromDB = cms.bool(False)
> -    process.PixelCPEGenericESProducer.useLAWidthFromDB = cms.bool(False)
> +    # use defaults d.k. 2/16
> +    #process.templates.DoLorentz=False
> +    #process.templates.LoadTemplatesFromDB = cms.bool(False)
> +    #process.PixelCPEGenericESProducer.useLAWidthFromDB = cms.bool(False)


I can really only speak for (phase 1) tracking, and we won't use the the customization alone without the era.

As I wrote earlier, I don't know what other parties are involved. I would guess the customize to be of interest only for pixel offline and tracking at the moment, but who knows... My personal opinion is that the customize will go away "soon", and if something is already migrated to eras it is probably not worth to update the corresponding part of the customize.

—
Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmssw/pull/13228/files#r52477843>.